### DISCUSSION TITLE
How the /fireball command exactly works

### DISCUSSION THREAD

**Discussion by AlexSnchez29** [EssentialsX/Essentials#4781] [Q&A] [answered]
I have tried to create the effect like /fireball in vanilla mixing the effect of the command /summon and /powertool, if I don't explain myself well, it is that I want to try to make a sword that shoots fireballs but with more power than those with the regular /fireball command, like 10-15, using the Nbt tag, specifically the {ExplosionPower:#}, but I can't find how to make the balls shoot in the direction I'm looking, if it's not possible to do it that's fine, I just wanted to try to shoot more powerful fireballs, maybe and this can be as a suggestion to increase the maximum usable power.

PS: Good plugin and keep up the good work :D

**Comment by osfanbuff63** [comment, marked as answer]
I think in vanilla, you could possibly use /execute align?

**Reply by AlexSnchez29** [reply]
I tried but that command is so complex, and I searched for a lot without results.
